src/components/views/elements/PersistentApp.js

```diff
@@ -82,6 +82,7 @@ export default class PersistentApp extends React.Component {
                     showDelete={false}
                     showMinimise={false}
                     miniMode={true}
+                    showMenubar={false}
```

Comment:
So this means that the user has no way of easily returning to the room the conference is in now, right?

Reply:
they never did - the menu bar doesn't provide this functionality